clippy_lints/src/new_without_default.rs

```diff
@@ -153,7 +153,7 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for NewWithoutDefault {
 }
 
 fn create_new_without_default_suggest_msg(ty: Ty) -> String {
-    #[rustfmt_skip]
+    #[cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
I think that should be `#[rustfmt::skip]` without the `cfg_attr`

Reply:
I also thought about it but decided to keep more consistent with current code base.
<details>
<summary>click here for `rg` output</summary>

```
$ rg "rustfmt" .     
pre_publish.sh
23:echo "remember to add a git tag and running 'cargo test' before committing the rustfmt changes"

tests/trim_multiline.rs
16:#[cfg_attr(rustfmt, rustfmt_skip)]
41:#[cfg_attr(rustfmt, rustfmt_skip)]

tests/needless_continue_helpers.rs
8:#[cfg_attr(rustfmt, rustfmt_skip)]
26:#[cfg_attr(rustfmt, rustfmt_skip)]
38:#[cfg_attr(rustfmt, rustfmt_skip)]
57:#[cfg_attr(rustfmt, rustfmt_skip)]
73:#[cfg_attr(rustfmt, rustfmt_skip)]

clippy_lints/src/new_without_default.rs
156:    #[cfg_attr(rustfmt, rustfmt_skip)]

clippy_lints/src/loops.rs
1781:#[cfg_attr(rustfmt, rustfmt_skip)]

clippy_lints/src/non_expressive_names.rs
91:#[cfg_attr(rustfmt, rustfmt_skip)]

clippy_lints/src/methods.rs
1824:#[cfg_attr(rustfmt, rustfmt_skip)]
1834:#[cfg_attr(rustfmt, rustfmt_skip)]
1868:#[cfg_attr(rustfmt, rustfmt_skip)]

clippy_lints/src/matches.rs
200:#[cfg_attr(rustfmt, rustfmt_skip)]

clippy_lints/src/lib.rs
211:#[cfg_attr(rustfmt, rustfmt_skip)]
```
</details>

Reply:
oh ^^